docs/version-2-upgrade.md

```diff
@@ -0,0 +1,385 @@
+Version 2.0.0 of the `runtime_connector` module now outputs a `sym_integration.runtime_context` by default. You may choose
+to use this integration instead of declaring one explicitly.
+
+For example, given this previous configuration:
+```terraform
+module "runtime_connector" {
+  source  = "symopsio/runtime-connector/aws"
+  version = "~> 1.0"
+
+  environment = var.environment  
+}
+
+resource "sym_integration" "runtime_context" {
+  type        = "permission_context"
+  name        = "runtime-${var.environment}"
+  external_id = module.runtime_connector.settings.account_id
+
+
+  settings = module.runtime_connector.settings
+}
+
+resource "sym_secrets" "this" {
+  type = "aws_secrets_manager"
+  name = "${var.environment}-secrets"
+
+  settings = {
+    context_id = sym_integration.runtime_context.id
+  }
+}
+```
+
+An updated configuration would be:
+```terraform
+module "runtime_connector" {
+  source  = "symopsio/runtime-connector/aws"
+  version = "~> 1.0"
+
+  environment = var.environment  
+}
+
+resource "sym_secrets" "this" {
+  type = "aws_secrets_manager"
+  name = "${var.environment}-secrets"
+
+  settings = {
+    context_id = module.runtime_connector.sym_integration.id
+  }
+}
+```
+
+Optionally, you may choose to use `moved` configuration blocks to migrate your Terraform state, instead of destroying and recreating your existing `sym_integration.runtime_context`:
+```terraform
+# This block may be removed after applying the updated configuration
+moved {  
+  from = sym_integration.runtime_context  
+  to   = module.runtime_connector.sym_integration.runtime_context  
+}
+```
+## New Output: `sym_runtime`
+Version 2.0.0 of the `runtime_connector` module now outputs a `sym_runtime.this` by default. You may choose
+to use this resource instead of declaring one explicitly.
+
+For example, given this previous configuration:
+```terraform
+module "runtime_connector" {
+  source  = "symopsio/runtime-connector/aws"
+  version = "~> 1.0"
+
+  environment = var.environment  
+}
+
+resource "sym_integration" "runtime_context" {
+  type        = "permission_context"
+  name        = "runtime-${var.environment}"
+  external_id = module.runtime_connector.settings.account_id
+
+
+  settings = module.runtime_connector.settings
+}
+
+resource "sym_runtime" "this" {
+  name       = var.environment
+  context_id = sym_integration.runtime_context.id
+}
+
+resource "sym_environment" "this" {
+  name       = var.environment
+  runtime_id = sym_runtime.this.id
+}
+```
+
+An updated configuration would be:
+```terraform
+module "runtime_connector" {
+  source  = "symopsio/runtime-connector/aws"
+  version = "~> 1.0"
+
+  environment = var.environment  
+}
+
+resource "sym_environment" "this" {
+  name       = var.environment
+  runtime_id = module.runtime_connector.sym_runtime.id
+}
+```
+
+Optionally, you may choose to use `moved` configuration blocks to migrate your Terraform state, instead of destroying and recreating your existing `sym_runtime`:
```

Comment:
```suggestion
Optionally, you may also use `moved` configuration blocks to migrate your Terraform state, instead of destroying and recreating your existing `sym_runtime`:
```